# Service used to delete a given map

string name
---
